integration-tests/test_manpage.py

```diff
@@ -0,0 +1,45 @@
+import gzip
+import pytest
+
+
+@pytest.mark.parametrize(
+    "option",
+    [
+        "--checkin",
+        "--compliance",
+        "--conf",
+        "--disable-schedule",
+        "--display-name",
+        "--enable-schedule",
+        "--group",
+        "--keep-archive",
+        "--list-specs",
+        "--logging-file",
+        "--net-debug",
+        "--no-upload",
+        "--offline",
+        "output-dir",
+        "--output-file",
+        "--quiet",
+        "--register",
+        "--force-reregister",
```

Comment:
`--force-reregister` was actually "neutered" in insights-core in December 2022 starting from 3.1.0, see https://github.com/RedHatInsights/insights-core/pull/3522

hence, since it actually always fail, I'd rather remove it from insights-core, and stop documenting it here